libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -1084,6 +1089,22 @@ private void setupLocationComponent(
     locationComponent.setRenderMode(RenderMode.COMPASS);
   }
 
+  /**
+   * Updates the size of the bounding box used to determine which route line was clicked. Upon
+   * a map click an attempt will be made to find a route line intersecting the bounding box. This
+   * is used to select alternative route lines on the map.
+   *
+   * Passing large integers values through this method means that a user doesn't
+   * have to be very accurate with tapping on an actual route line because the invisible
+   * box is large and thus, there's a good chance that a portion of a route line will run
+   * through the query box's area. A smaller rectangle will decrease the chances.
+   */
+  public void updateClickDistancePadding(int newClickDistancePadding) {
+    if (mapRoute != null) {
+      mapRoute.updateClickDistancePadding(newClickDistancePadding);
+    }
+  }
```

Comment:
Does this ever need to be adjusted dynamically? Would an immutable option in the builder be enough?